content/introduction-and-setup/walkthrough/windows-install/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Windows Users: Visual Studio Community Edition"
+date: 2022-12-15T09:16:07-06:00
+draft: false
+weight: 2
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Before you start the installation guide, read through each step below. You will find the recommended features and functionality to install listed in each step.
+
+{{% notice red "Warning" "rocket" %}}
+Allow at least an hour for the installation process.
+{{% /notice %}}
+
+## Installation Steps
+
+Walk through the installation process by following Microsoft’s [Install Visual Studio](https://learn.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2022) guide.
+
+1. Verify that your computer can run Visual Studio 2022.
+1. Download the *Community* version of Visual Studio 2022.
+
+   {{% notice orange "Note" "rocket" %}}
+   Make sure you select Visual Studio 2022.
+   You will see multiple options for software downloads, such as Visual Studio Code and Visual Studio for Mac, on this page. Do NOT select either of these options!
+   {{% /notice %}}
+
+1. Install the Visual Studio Installer
+1. Chose your **Workloads**:
+
+   1. Select the following packages from the Workloads pane:
+
+      1. ASP.NET and web development
+      1. Azure development
+      1. Data storage and processing
+      1. .NET core cross-platform development
```

Comment:
Nested lists should follow the following formatting convention:

1.
    a.
        i.